lib/node_modules/@stdlib/math/base/special/sincos/lib/main.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+// MODULES //
+
+var computeSincos = require( './sincos.js' );
+
+
+// MAIN //
+
+/**
+* Simultaneously computes the sine and cosine of a number.
+*
+* @param {array} [out] - destination array
```

Comment:
P.S. Besides these standard JSDoc type annotations, you might want to have a look at our custom type definitions, which live here: https://github.com/stdlib-js/stdlib/tree/develop/tools/docs/jsdoc/typedefs